### DISCUSSION TITLE
Email notifications

### DISCUSSION THREAD

**Discussion by vg67000** [immich-app/immich#12687] [General]
Hello,

Here are an idea for the email notification functionality:

- It would be nice to be able to have a list of external emails (without account on the immich server) that would receive notification when new photos are uploaded to an album (attached to a sharing link for example, and so with the public link in the email so that people can directly click in the email). 

Regards

**Comment by radh21301** [comment]
same as :
https://github.com/immich-app/immich/discussions/10508
https://github.com/immich-app/immich/issues/12424

This PR has the fix https://github.com/immich-app/immich/pull/12223. Keep an eye out.

**Reply by vg67000** [reply]
Thanks, I didn't see those, I edited to keep only the other point.
Very nice to see that the development of the app is going so fast !
